ApiCalls.js

```diff
@@ -0,0 +1,44 @@
+const getUserFlights = () => {
+  return fetch('https://fa654a41-d040-4528-99c2-4e7b349236b9.mock.pstmn.io/user/1/flights')
+    .then(response => {
+      if (!response.ok) {
```

Comment:
This is a great start, we can always refactor with async await at a later point.